template.go

```diff
@@ -386,7 +386,6 @@ const (
 					{{- range $method := $.Interface.Methods }}
 						m.Minimock{{$method.Name}}Inspect()
 					{{ end -}}
-					m.t.FailNow()
```

Comment:
I don't think this should be removed, we should call `m.t.Fail()` instead to ensure that the test is still marked as failed. 

It also requires a modification of the `Tester` interface

Reply:
There is no need to call Fail here, because in m.Minimock{{$method.Name}}Inspect() one of Errorf functions are already executed by the time we get here, which call Fail inside